frontend/src/views/InterpreterDirectory.tsx

```diff
@@ -30,10 +31,15 @@ export default function InterpreterDirectory () {
 
   const getSearchResults = async (params: InterpreterSearchParams) => {
     setSearch(params)
+    // Changing query options as per user selection
+    const query: InterpreterSearchParams = { 
+      ... params,
+      active: params.active === 'Active' ? 'TRUE' : 'FALSE',
```

Comment:
Why not pass these as `true` or `false`?

Reply:
Raw query generator utility unable to generate query for boolean value (specially for case false), but works fine if I supply this as string.